packages/backend/src/payment-method/ilp/service.test.ts

```diff
@@ -681,41 +697,5 @@ describe('IlpPaymentService', (): void => {
         expect((error as PaymentMethodHandlerError).retryable).toBe(false)
       }
     })
-
-    test('throws if invalid quote data', async (): Promise<void> => {
```

Comment:
we don't use `additionalFields` so parsing JSON is not required, so this test is not necessary